### DISCUSSION TITLE
Is there a "Send To Amazon" API fix pending, or is the current situation more permanent?

### DISCUSSION THREAD

**Discussion by johnkw** [amzn/selling-partner-api-models#3957] [General]
Is there a "Send To Amazon" API fix pending, or is the current situation more permanent?

Either way this needs to be documented in https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v0-use-case-guide because currently the situation is bizarre in that API shipments are not easy to view in Seller Central during certain situations.

Once the Shipment ships or arrives, it becomes fairly normal.  But prior to that Seller Central acts like API shipments are something weird that needs to be fixed.

**Comment by daviddecorso** [comment]
It's like they don't want us to use the API

**Comment by shreeharsh-a** [comment]
Please reach out to [developer support](https://developer.amazonservices.com/support) so that we can better help you with issues related to SP API.

`Note: SP API or docs related issues / troubleshooting support is managed by a different team. We can help if the issue is related to the content published on this repo.`

**Comment by johnkw** [comment]
This is still an open bug that deserves to remain open -- can you convert this to a discussion, since that's where open bugs go now apparently?

**Comment by johnkw** [comment]
Has anyone made it past bug #556 in order to see if this weirdness still happens with the new API?

**Comment by MeenaAmz** [comment]
Can you please provide more details on the specific issue with Send to Amazon?

**Comment by johnkw** [comment]
In the Seller Central website after you create a shipment with the API, the web page says you can't view the shipment because it was created via the API.

**Reply by MeenaAmz** [reply]
Was the shipment created via the new API [v2024-03-20](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference) or [v0](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v0-reference)? If it was created using v0, interoperability with Seller Central (send to amazon) will be limited. The new version provides more interoperability.

**Reply by johnkw** [reply]
Shipments are only possible with v0 right now.  As per bug #556, v2024-03-20 is not yet fully available.

That's good to hear though that once v2024-03-20 is available for use, then this bug regarding Seller Central will hopefully go away.
